charts/pipelines-library/scripts/tekton-prune.sh

```diff
@@ -79,51 +62,29 @@ prune_resources() {
 main() {
   separator=';'
   pvc_owner_kind='PipelineRun'
-  pods_to_delete_file_path='/tmp/pods-to-delete.txt'
-  pvcs_to_delete_file_path='/tmp/PVCs-to-delete.txt'
+  pipelinerun_to_delete_file_path='/tmp/runs-to-delete.txt'
 
   verify
 
   echo 'Get active pipelineruns'
   active_pipelineruns=$(get_active_pipelineruns)
-  echo "active pipelineruns: $active_pipelineruns"
+  echo "Running pipelineruns: $active_pipelineruns"
 
-  echo "Get pipelineruns completed recently (in the last ${RECENT_MINUTES_PODS} minutes)"
-  recent_pipelineruns_pods=$(get_recent_pipelineruns "${RECENT_MINUTES_PODS}")
-  echo "recent pipelineruns: $recent_pipelineruns_pods"
+  echo "Get pipelineruns completed recently (in the last ${RECENT_MINUTES_LOGS} minutes)"
+  echo "Pipelineruns completed recently: $recent_pipelineruns"
```

Comment:
recent_pipelineruns - what does this variable contain?

Reply:
removed